clap_derive/src/utils/doc_comments.rs

```diff
@@ -78,14 +78,14 @@ fn split_paragraphs(lines: &[&str]) -> Vec<String> {
         let len = slice
             .iter()
             .position(|s| is_blank(s))
-            .unwrap_or_else(|| slice.len());
+            .unwrap_or(slice.len());
```

Comment:
Why is clippy allowing this?  It normally wants function calls in an `or_else`

Reply:
Yeah this one caused me to pause also. I'll do some digging. Possibly a false positive

Reply:
Wait, it's a slice isn't it? Slice::len is `const`

Reply:
All being `const` means is that if all inputs are known at compile time, then this can be evaluated at compile time.  The input isn't known at compile time, so a function call is being made.

Either way, I'm assuming the `len()` function call is being compiled away anyways so `unwrap_or` has no performance penalty compared to `unwrap_or_else`.  Just seems strange.